Clicker/Views/Cards/FROptionCell.swift

```diff
@@ -111,27 +111,27 @@ class FROptionCell: UICollectionViewCell {
         self.didUpvote = frOptionModel.didUpvote
         self.answerId = frOptionModel.answerId
         optionLabel.text = frOptionModel.option
-        let numUpvotedLabelTextColor: UIColor = frOptionModel.didUpvote ? .clickerBlue : .clickerGrey2
+        let numUpvotedLabelTextColor: UIColor = frOptionModel.didUpvote ? .clickerGreen0 : .clickerGrey2
         numUpvotedLabel.textColor = numUpvotedLabelTextColor
         numUpvotedLabel.text = "\(frOptionModel.numUpvoted)"
-        let upvoteImageViewImage = frOptionModel.didUpvote ? #imageLiteral(resourceName: "blueTriangle") : #imageLiteral(resourceName: "greyTriangle")
+        let upvoteImageViewImage = frOptionModel.didUpvote ? #imageLiteral(resourceName: "upvote-green") : #imageLiteral(resourceName: "upvote-grey")
         upvoteImageView.image = upvoteImageViewImage
     }
 
     // MARK: - Updates
     func update(with frOptionModel: FROptionModel) {
         self.didUpvote = frOptionModel.didUpvote
         optionLabel.text = frOptionModel.option
-        let numUpvotedLabelTextColor: UIColor = frOptionModel.didUpvote ? .clickerBlue : .clickerGrey2
+        let numUpvotedLabelTextColor: UIColor = frOptionModel.didUpvote ? .clickerGreen0 : .clickerGrey2
         numUpvotedLabel.textColor = numUpvotedLabelTextColor
         numUpvotedLabel.text = "\(frOptionModel.numUpvoted)"
-        let upvoteImageViewImage = frOptionModel.didUpvote ? #imageLiteral(resourceName: "blueTriangle") : #imageLiteral(resourceName: "greyTriangle")
+        let upvoteImageViewImage = frOptionModel.didUpvote ? #imageLiteral(resourceName: "upvote-green") : #imageLiteral(resourceName: "upvote-grey")
```

Comment:
Can you define a constant for the image resource names here?

Reply:
Since it's an image literal, XCode displays this code as an image in a box, I think this is how GitHub displays a literal. In the project, 'resourceName' property doesn't exist.

Reply:
Right, that makes sense. Thanks!

Reply:
would it be possible to use `UIImage(named:)` instead? i'm not a huge image literals